extensions/spring-data-jpa/deployment/src/test/java/io/quarkus/spring/data/deployment/BasicTypeDataRepositoryTest.java

```diff
@@ -75,6 +79,33 @@ public void testTimeZonesByLocale() {
         assertThat(timeZones).isNotEmpty().contains(TimeZone.getTimeZone("CST"));
     }
 
+    @Test
+    @Order(5)
+    @Transactional
+    public void testMapInterfacesUsingList() {
+        List<WithDoubleValue> list = repo.findAllDoubleValuesToList();
+        Set<Double> actual = list.stream().map(WithDoubleValue::getDoubleValue).collect(Collectors.toSet());
+        assertThat(actual).isNotEmpty().contains(Math.PI);
+    }
+
+    @Test
+    @Order(6)
+    @Transactional
+    public void testMapInterfacesUsingPage() {
+        Page<WithDoubleValue> list = repo.findAllDoubleValuesToPage();
+        Set<Double> actual = list.stream().map(WithDoubleValue::getDoubleValue).collect(Collectors.toSet());
+        assertThat(actual).isNotEmpty().contains(Math.PI);
+    }
```

Comment:
What about testing `getTotalElements()` etc.? Same for the `Slice` test method.

Reply:
Done